src/cargo/core/workspace.rs

```diff
@@ -46,6 +46,9 @@ pub struct Workspace<'cfg> {
     // set above.
     members: Vec<PathBuf>,
 
+    // The subset of `members` that are “default”.
```

Comment:
Let's expand this comment a bit, to explain when exactly this field is used?

Reply:
I wrote some things.